docs/reference/announcements.md

```diff
@@ -108,6 +108,10 @@ An example of how to use the new `CorrelationResult` can be found in the [Connec
 
 The separated Ingress Helm configuration for Camunda 8 Self-Managed has been deprecated in 8.6, and will be removed from the Helm chart in 8.7. Only the combined Ingress configuration is officially supported. See the [Ingress guide](/self-managed/setup/guides/ingress-setup.md) for more information on configuring a combined Ingress setup.
 
+#### Helm chart - `global.multiregion.installationType` deprecation
+
+The `global.multiregion.installationType` option is used in failover and failback scenarios. This option in the Helm chart has been deprecated in 8.6, and will be removed from the Helm chart in 8.7. `global.multiregion.installationType` will be replaced with a set of API endpoints called while following the ([dual-region operational procdure](/self-managed/operational-guides/multi-region/dual-region-ops.md))
```

Comment:
English is not my first language and I'm fine keeping it as is.

I'm just wondering whether it should be
```
`global.multiregion.installationType` will be replaced with
```

or

```
`global.multiregion.installationType` was replaced with
```

The later suggests that the change has already happened while the other may suggest that something may be yet to come.

Reply:
Maybe @conceptualshark from a technical writer review.
Again, I'm fine keeping as is, not sure what's the best phrasing.

Overall, great work 🚀

Reply:
If it's the case that it's already replaced and should be used in 8.6, I support updating `will be` to `was` as well.

Reply:
@jessesimpson36, I added the adjustment in the commit: https://github.com/camunda/camunda-docs/pull/4274/commits/d6b9db64cf63c4f1471e3bd5bca264a7dfcc3d17 so we can merge this PR.